Source/DafnyDriver/Commands/MeasureComplexityCommand.cs

```diff
@@ -24,12 +28,17 @@ static class MeasureComplexityCommand {
 
     DooFile.RegisterNoChecksNeeded(Iterations, false);
     DooFile.RegisterNoChecksNeeded(RandomSeed, false);
+    DooFile.RegisterNoChecksNeeded(TopX, false);
   }
 
+  private static readonly Option<uint> TopX = new("--top-x", () => 10U,
```

Comment:
I feel like there must be a better name for this. But I haven't been able to come up with one yet!

Reply:
I changed it to `--worst-amount`